index.html

```diff
@@ -414,37 +404,50 @@ <h2>Attributes</h2>
               invoked.</p>
             </li>
             <li>
-              <p>Let <var>delay</var> be the argument to the setter.</p>
+              <p>Let <var>depth</var> be the argument to the setter.</p>
+            </li>
+            <li>
+              <p>If <var>depth</var> is negative or larger than 4000 milliseconds, then
+              [=exception/throw=] a {{RangeError}}.</p>
+            </li>
+            <li>
+              <p>Set <var>receiver</var>'s {{RTCRtpReceiver/[[JitterBufferDepth]]}}
+              to <var>depth</var>.</p>
             </li>
             <li>
-              <p>If <var>delay</var> is negative or larger than 4 seconds then,
-              [=exception/throw=] a {{RangeError}} and abort these steps.</p>
+              <p>If <var>depth</var> is below <a>minimum allowed
+              depth</a> or above <a>maximum allowed depth</a> then clamp
+              <var>depth</var> to <a>minimum allowed
+              depth</a> or <a>maximum allowed depth</a> to be as close as
+              possible to the original value.</p>
```

Comment:
Since min and max is something that changes continuously, it would be wrong to apply the clamping as a one-time event. If min is 1 ms most of the time, but then there is a sudden glitch and min is momentarely set to 1000 ms, but the user sets jitterBufferDepth to 50 ms, I would expect the jitter buffer target to be as close to 50 ms as possible as soon as "min" has recovered, and not forever be stuck at 1000 ms as this is not what the app asked for.

Also doing this in the JS thread doesn't make sense since the jitter buffer min and max values would adjusted by the user agent in parallel.

I have a suggested edit below to replace this paragraph.

Reply:
Yes, me moving these out was indeed based on my assumption they were constant. Note that even [in parallel](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel) steps _"are to be run, one after another"_ in sequence, so this wasn't clear before. I know we sometimes use a shorthand to describe future reactions inside steps, when _"When X happens, do this"_ seems too much.